fixtures/external-durable-objects-app/package.json

```diff
@@ -5,9 +5,9 @@
 	"scripts": {
 		"check:type": "tsc",
 		"dev": "npx concurrently -s first -k \"wrangler dev a/index.ts --local --port 8400\" \"wrangler dev b/index.ts --local --port 8401\" \"npx wrangler dev c/index.ts --local --port 8402\" \"cd d && npx wrangler pages dev public --port 8403 --do PAGES_REFERENCED_DO=MyDurableObject@a\"",
-		"test": "npx vitest run",
-		"test:ci": "npx vitest run",
-		"test:watch": "npx vitest",
+		"test": "vitest run",
+		"test:ci": "vitest run",
```

Comment:
here and for all other fixtures etc... aren't the `test:ci` scripts useless now? shouldn't we remove them?